<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDiagramDialog</class>
 <widget class="QDialog" name="ExportDiagramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccione formato de exportación:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Imágen
(*.png)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonExcel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Archivo
Excel
(*.xlsx)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCSV">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Archivo CSV
(*.csv)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
